File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Java/Exercises/37_compito_java_2020_01_24/Campo.java
public class Campo {
    protected int codice;
    protected String sport, nome;
    protected float larghezza, lunghezza, costo;
    protected int prenotazioni;

    public Campo(int codice, String sport, String nome, float larghezza, float lunghezza, float costo) {
        this.codice = codice;
        this.sport = sport;
        this.nome = nome;
        this.larghezza = larghezza;
        this.lunghezza = lunghezza;
        this.costo = costo;
        prenotazioni = 0;
    }
    
    public void addPrenotazione() {
    	prenotazioni++;
    }
    
    public float incasso() {
    	return prenotazioni*costo;
    }
    
    public String toString() {
    	return nome + "\t" + codice + "\t" + larghezza + "\t" + lunghezza + "\t";
    }
}

File: Programming-languages-and-laboratory/Python/Lessons/26_20221202/37_soluzione/Campo.py



class Campo:
    def __init__(self, codice, nome,  larghezza,  lunghezza,   costo):
        self._codice = codice
        self._nome=nome
        self._larghezza = larghezza
        self._lunghezza = lunghezza
        self._costo=costo
        self._prenotazioni = 0

    def addPrenotazione(self):
    	self._prenotazioni=self._prenotazioni+1;	
    def incasso(self):
    	return self._prenotazioni*self._costo;
 
    def __str__(self):
        return self._nome+"\t"+str(self._codice) + "\t" + str(self._larghezza)+"\t"+str(self._lunghezza)+"\t"
